start.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e
+
+python3 init.py
+exec fastapi run app/main.py "$@"
```

Comment:
Use environment variables instead of command arguments
```suggestion
exec fastapi run app/main.py --workers ${NB_WORKERS:1}
```